test/dynamo/test_optimizations.py

```diff
@@ -166,11 +165,15 @@ def test_ipex_fp32(self):
         model = model.eval()
         input = torch.randn(8, 3, 64, 64).contiguous(memory_format=torch.channels_last)
         r1 = model(input)
-        opt_model = torch._dynamo.optimize(backends.ipex_fp32)(model)
-        with torch.no_grad():
-            r2 = opt_model(input)
-        self.assertTrue(same(r1, r2))
-        self.assertEqual(r2.dtype, torch.float32)
+        for dynamic_shapes in [True, False]:
+            torch._dynamo.reset()
+            torch._dynamo.config.dynamic_shapes = dynamic_shapes
```

Comment:
Can you use `with patch` to make sure the configuration changes here won't impact others?

Reply:
Done.